### PR TITLE
Add genre info

### PR DESCRIPTION
**Pull request by AleksandroBoko**
There was added opportunity to show and choose genre of the games.

### CODE REVIEW

File: GameStore.Services/Services/IGenreService.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using GameStore.Domains.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameStore.Services.Services
+{
+    public interface IGenreService:IService<GenreModel>
```
Comment (SafeNuts) [inline GameStore.Services/Services/IGenreService.cs]: Don't forget about white spaces

File: GameStore.Services/Services/Implementation/GenreServices.cs [hunk 2]
```diff
@@ -47,6 +47,19 @@ public GenreModel GetItemById(Guid id)
             }
         }
 
+        public GenreInfoTransferModel GetGenreInfoTransferById(Guid id)
+        {
+            var genre = genreRepository.GetItemById(id);
+            if (genre == null)
+            {
```
Comment (SafeNuts) [inline GameStore.Services/Services/Implementation/GenreServices.cs]: Let's try to implement this as ternary operator

File: GameStore/Controllers/MainController.cs [hunk 3]
```diff
@@ -11,21 +11,39 @@ namespace GameStore.Controllers
...
 
-        //public IEnumerable<>
+        [Route("api/main/genres")]
+        public IEnumerable<GenreModel> GetGenres() => genreService.GetAll();
+
+        [Route("api/main/genregames/{id}")]
+        public ICollection<GameRateTransferModel> GetGenreGames(Guid id)
+        {
+            var genre = genreService.GetGenreInfoTransferById(id);
+            if (genre != null)
```
Comment (SafeNuts) [inline GameStore/Controllers/MainController.cs]: Let's try to implement this as ternary operator

File: GameStore.Services/Services/IGenreService.cs (hunk 1 above)
Comment (AleksandroBoko) [reply GameStore.Services/Services/IGenreService.cs]: FIxed

File: GameStore.Services/Services/Implementation/GenreServices.cs (hunk 2 above)
Comment (AleksandroBoko) [reply GameStore.Services/Services/Implementation/GenreServices.cs]: Changed condition operator

File: GameStore/Controllers/MainController.cs (hunk 3 above)
Comment (AleksandroBoko) [reply GameStore/Controllers/MainController.cs]: Changed condition operator

**Pull request merged**